src/_P112_AS7265x.ino

```diff
@@ -165,7 +165,19 @@ boolean Plugin_112(byte function, struct EventStruct *event, String& string)
       }
       break;
     }
+    case PLUGIN_TEN_PER_SECOND:
+    {
+      P112_data_struct *P112_data =
+        static_cast<P112_data_struct *>(getPluginTaskData(event->TaskIndex));
 
+      if (nullptr != P112_data) {
+        if (P112_data->sensor.dataAvailable()) {
+          // Measurement was succesfull, schedule a read.
+          Scheduler.schedule_task_device_timer(event->TaskIndex, millis() + 10);

```

Comment:
Not sure what may happen if `PLUGIN_TEN_PER_SECOND` is called again before `PLUGIN_READ` is called.
So maybe also add a flag in the `P112_data` struct whether or not we're waiting for data.
Set it when starting a measurement and clear it in this code.